benches/bench.rs

```diff
@@ -1055,7 +1146,13 @@ fn bench_lookup_function(c: &mut Criterion) {
 
     let contract_identifier = QualifiedContractIdentifier::local(&*format!("c{}", 0)).unwrap();
     let mut contract_context = ContractContext::new(contract_identifier.clone());
-    let (_, contract) = gen(function, SCALE, SCALE * 2);
+
+    let GenOutput {
+        setup: _,
+        body: contract,
+        input_size: _,
+    } = gen(function, SCALE, SCALE * 2);
```

Comment:
LookupFunction should take 1 as input size, and you can change the following line in `generators.rs`:
```
ClarityCostFunction::LookupFunction => gen_ast_cycle_detection(input_size),
```
to 
```
ClarityCostFunction::LookupFunction => gen_ast_cycle_detection(scale),
```